src/Explorer/DataSamples/ContainerSampleGenerator.test.ts

```diff
@@ -137,7 +137,13 @@ describe("ContainerSampleGenerator", () => {
   it("should not create any sample for Table API account", async () => {
     const experience = "not supported api";
     const explorerStub = createExplorerStub(undefined);
-    explorerStub.isPreferredApiTable = ko.computed<boolean>(() => true);
+    updateUserContext({
+      databaseAccount: {
+        properties: {
+          capabilities: [{ name: "EnableTable" }],
+        },
+      } as DatabaseAccount,
+    });
```

Comment:
@southpolesteve, this bugs me a bit: because of the way `userContext` is, and because of the way it is updated, this test can't just do `updateUserContext({ apiType: 'flef' })`, so it ends up having to inadvertently exercise and test  `updateUserContext()` itself.

I don't know what the solution is though. It wouldn't bother me much if it was just this test, but because it's `userContext`, it has the potential to spread out fast.

Reply:
Discussed with Jordi offline. This bugs me too. But I think we need to tackle this problem with `userContext` in a separate step. Getting out from under Explorer is more important IMO and this PR helps.